### ISSUE TITLE
DoubleClick doesn't work in SmartGWT framework

### ISSUE THREAD

**Issue by FibYar** [SeleniumHQ/selenium#916]
I'm using Selenium WebDriver to write tests on application based on SmartGWT. It almost works, but there is a problem with doubleClick(). It works like single click (but it works on html-based applications).
I'm using selenium-java 2.47.0.
Sample code:
```
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
        driver.get("http://www.smartclient.com/smartgwt/showcase/#grid_summaries_featured_category");
        WebElement webElement = driver.findElement(By.xpath("//*[@class='cell']//*[text()='Office Application Suite']"));
        Actions actions = new Actions(driver);
        actions.doubleClick(webElement).perform();
```
Expected result: a line in grid should transform to input fields (you can try to do this by hands, not Selenium).
![screenshot 002](https://cloud.githubusercontent.com/assets/12131583/9222461/9a59ed9c-40fa-11e5-8858-52a1919bffdc.png)

Actual result: a line doesn't transform to input fields (it's like a single click).
![screenshot 003](https://cloud.githubusercontent.com/assets/12131583/9222469/aa4e13f4-40fa-11e5-93e3-779013ea9b8d.png)

**Comment by lukeis** [comment]
Is your issue only with Chrome? If so, please log a bug on that project:
https://code.google.com/p/chromedriver/issues/list

If it's with FirefoxDriver or InternetExplorerDriver or SafariDriver. please try there and let us know if there's an issue with them.

**Comment by FibYar** [comment]
The problem reproduces both in ChromeDriver and FirefoxDriver.

**Comment by barancev** [comment]
1) You need to perform the following actions:
actions.click(webElement).doubleClick(webElement).perform();

2) Fixed in Firefox by commit 7c89bbddebf8e40d278d5e00e1b0df270e94b3dc , the fix will be available in 2.48
